pandas/plotting/_core.py

```diff
@@ -1131,7 +1131,7 @@ def line(self, x=None, y=None, **kwargs) -> PlotAccessor:
     )
     @Substitution(kind="bar")
     @Appender(_bar_or_line_doc)
-    def bar(self, x=None, y=None, **kwargs) -> PlotAccessor:
+    def barplot(self, x=None, y=None, **kwargs) -> PlotAccessor:
```

Comment:
Same comment as on other usage, can't change the name since public API. Could disable or might also be reasonable to modify the `bad-names` config for this rule and remove `bar` (https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#bad-names)

Reply:
thx, I had already disabled, but removing bar from the config also seems reasonable